runtime/oti/j9nonbuilder.h

```diff
@@ -5838,8 +5844,8 @@ typedef struct J9JavaVM {
 #endif /* (JAVA_SPEC_VERSION >= 16) || defined(J9VM_OPT_CRIU_SUPPORT) */
 #define J9JAVAVM_REFERENCE_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(U_32) : sizeof(UDATA))
 #define J9JAVAVM_OBJECT_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9ObjectCompressed) : sizeof(J9ObjectFull))
-#define J9JAVAVM_CONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectContiguousCompressed) : sizeof(J9IndexableObjectContiguousFull))
-#define J9JAVAVM_DISCONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectDiscontiguousCompressed) : sizeof(J9IndexableObjectDiscontiguousFull))
+#define J9JAVAVM_CONTIGUOUS_HEADER_SIZE(vm) (vm->sizeOfJ9IndexableObjectContiguous)
+#define J9JAVAVM_DISCONTIGUOUS_HEADER_SIZE(vm) (vm->sizeOfJ9IndexableObjectDiscontiguous)
```

Comment:
Questionable if these are even needed anymore. They aren't even descriptive enough (there is not even Indexable in the name). Check how much they are used and decide for yourself.

Reply:
Ok. They are used in a few places but I will replace their uses. I am also realizing that my field names could be more descriptive. I will refactor my field name now as well.